MakeHist/MET.C

```diff
@@ -99,4 +126,101 @@ void MET(){
 
     leg_MET->Draw();
 
+    c1->SaveAs("./hists/"+prefix+"_MET.png");
+
+     //make bkg for siginificance plot
+    MET_DB->Add(MET_ST);
+    MET_DB->Add(MET_TT);
+    MET_DB->Add(MET_DY);
+
+    TH1F * MET_DB_copy = (TH1F*) MET_DB->Clone("DB_copy");
+
+    TH1F * MET_200_copy = (TH1F*) MET_200->Clone("200_copy");
+
+    //std::cout << "ComputeIntegral " << MET_DB->ComputeIntegral() << "\n";
+    float previousBin;
+    float currentBin;
+    int nBins = MET_DB->GetNbinsX();
+
+    //std::cout << "right cut \n";
+    for(int i=-2;i<nBins+3;i++){
```

Comment:
Basically, same comments as in HTLT file. Moreover, you can define a class, calculating significance vs selection cut for any distribution and avoid code duplications